deps/docker-compose.yml

```diff
@@ -24,7 +25,10 @@ services:
 volumes:
   db-data-api:
     driver: local
+  state-pgadmin:
+    driver: local
 
 networks:
   qkm:
-    driver: bridge
\ No newline at end of file
+    external:
```

Comment:
please remove all the changes of this file

Reply:
ok